spec/factories/factories.rb

```diff
@@ -45,4 +45,11 @@
       n
     end
   end
+
+  factory :project do
+    repo_url 'https://github.com/user/repo'
+    live_preview 'http://mysite.com'
+    association :user
```

Comment:
I'd avoid putting associations in the factory by default as this slows down tests significantly. We can pass associations in when we use the factory in a test